### ISSUE TITLE
Shouldn't this error message to be rewritten?

### ISSUE THREAD

**Issue by byassine52** [nicklockwood/SwiftFormat#616]
I got this error when specifying tab usage in `.swiftformat`:
```
...
--indent tab
...
```

And I got this error message:
```
error: The --maxwidth option requires --tabwidth to also be set.
```

Shouldn't this error message to be rewritten?
It took me a while to find out that I needed to set `--tabwidth` value, and `--maxwidth` had nothing to do there.

https://github.com/nicklockwood/SwiftFormat/blob/9508a160654f03ce8ba6275efacdb160181c21d7/Sources/CommandLine.swift#L793

**Comment by nicklockwood** [comment]
@byassine52 good point. I'll fix it.

**Comment by nicklockwood** [comment]
Fixed in 0.44.8
